projects/js-packages/components/changelog/remove-threat-components

```diff
@@ -0,0 +1,4 @@
+Significance: major
```

Comment:
Using major here as we are removing components. Is this necessary?

Reply:
It is! Thanks!